assets/loc/translations.csv

```diff
@@ -466,12 +481,14 @@ Power,Power,파워,パワー,Poder
 Incoming {SpecialStr}Damage: {Amount}{PreventStr},Incoming {SpecialStr}Damage: {Amount}{PreventStr},Incoming {SpecialStr}Damage: {Amount}{PreventStr},受ける{SpecialStr}ダメージ: {Amount}{PreventStr},Entrante {SpecialStr}Daño: {Amount}{PreventStr}
 Choose the next die result.,Choose the next die result.,다음에 던졌을 때 나올 주사위 눈을 선택하세요.,次のサイコロの結果を選んでください。,Elije el siguiente resultado del dado:
 Oshi:,Oshi:,오시:,推し:,Oshi:
+ACTION_MENU__SUPPORT,Support:,서포트:,サポート:,Apoyo:
```

Comment:
I'm fine with leaving this as "Support:" for the current special action. If they come from other places maybe we'll add those then.